src/tree/CosmosDBAccountWizard/CosmosDBAccountCapacityStep.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { localize } from '../../utils/localize';
+import { ICosmosDBWizardContext } from './ICosmosDBWizardContext';
+
+export class CosmosDBAccountCapacityStep extends AzureWizardPromptStep<ICosmosDBWizardContext> {
+
+    public async prompt(wizardContext: ICosmosDBWizardContext): Promise<void> {
+
+        const placeHolder: string = localize('selectDBServerMsg', 'Select a capacity model.')
```

Comment:
Not sure if we have to go with what the portal says, but FWIW

> Choose the capacity mode for your database operations.

Reply:
I think I would prefer "Select a capacity model." as it would consistent with other placeholders. 
The "Choose the capacity mode..." might confuse the user as capacity model is selected at the server even tho it is meant for database operations. 

What if we add a "Learn More..." option to this prompt instead?